tests/functional/baseline-no-breadcrumbs/collections/ns/col2/index.rst

```diff
@@ -35,6 +35,7 @@ Description
     :maxdepth: 1
 
 
+
```

Comment:
what's this extra line for?

Reply:
This comes from the empty line between:
```jinja2
{% endif %}

{% for section in extra_docs_sections %}
```

Reply:
Removed in cb2c5873b542650208a79aa19922346d40ae6ad6.